assets/stylesheets/layout/_main.scss

```diff
@@ -375,10 +376,56 @@ $sidebar-width-min: 228px;
 	}
 }
 
+.focus-design {
+	.wp-primary {
+		opacity: 0.2;
+		pointer-events: none;
+	}
+
+	.wp-secondary .design-menu {
+		opacity: 1;
+		transform: translateX( 272px );
+		pointer-events: auto;
+	}
+
+	.sidebar,
+	.wp-secondary .site-selector {
+		pointer-events: none;
+	}
+}
+
 .focus-sidebar {
 	overflow: hidden;
 }
 
+.design-menu {
```

Comment:
I put this here for the prototype, but we should move it to `my-sites/design-menu/style.scss`